cmd/transfer-pvc/transfer-pvc.go

```diff
@@ -80,9 +87,14 @@ func NewTransferOptions(streams genericclioptions.IOStreams) *cobra.Command {
 
 func addFlagsForTransferPVCOptions(t *TransferPVCOptions, cmd *cobra.Command) {
 	cmd.Flags().StringVar(&t.SourceContext, "source-context", "", "The name of the source context in current kubeconfig")
-	cmd.Flags().StringVar(&t.DestinationContext, "destination-context", "", "The name of destination context current kubeconfig")
-	cmd.Flags().StringVar(&t.PVCNamespace, "pvc-namespace", "", "The namespace of the pvc which is to be transferred, if empty it will try to use the namespace in source-context, if both are empty it will error")
-	cmd.Flags().StringVar(&t.PVCName, "pvc-name", "", "The pvc name which is to be transferred on the source")
+	cmd.Flags().StringVar(&t.DestinationContext, "destination-context", "", "The name of destination context in current kubeconfig. Will use the default context")
+	cmd.Flags().StringVar(&t.SourcePVCNamespace, "source-pvc-namespace", "", "The namespace of the pvc which is to be transferred, if empty it will try to use the namespace in source-context, if both are empty it will error")
+	cmd.Flags().StringVar(&t.SourcePVCName, "source-pvc-name", "", "The pvc name which is to be transferred on the source")
+	cmd.Flags().StringVar(&t.DestinationPVCNamespace, "destination-pvc-namespace", "", "The namespace of the pvc which is to be transferred, if empty it will try to use the namespace in destination-context, if both are empty it will error")
+	cmd.Flags().StringVar(&t.DestinationPVCName, "destination-pvc-name", "", "The pvc name which is to be transferred on the source")
```

Comment:
can we change the help tool tip here?

```
"The pvc name which is to which the source data will be transferred to. If the name is empty, it will be the same as the source name"
```